crypto/wolfcrypt/shim/rubin_wc_shim.c

```
@@ -25,6 +25,33 @@
 #include <stdint.h>
 #include <string.h>
 
+#define RUBIN_U32_MAX ((size_t)0xffffffffu)
+
+#define RUBIN_ML_DSA87_PUBKEY_BYTES 2592u
+#define RUBIN_ML_DSA87_SIG_BYTES 4627u
+#define RUBIN_SLH_DSA_SHAKE_256F_PUBKEY_BYTES 64u
+#define RUBIN_SLH_DSA_SHAKE_256F_SIG_BYTES 49856u
+
+/* Error codes for signature verify APIs (negative => failure). */
+#define RUBIN_WC_ERR_SHA3_INPUT_TOO_LARGE -6
```

Clang (14.0.6)

[warning] macro replacement list should be enclosed in parentheses

(bugprone-macro-parentheses)